### ISSUE TITLE
The language dropdown menu does not have the correct behavior

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#1428]
## Bug Report

**Problematic behavior**
The language dropdown menu behaves like a button. This is confusing for users of assistive technologies, as the behavior is not the same as a standard dropdown menu.

**Expected behavior/code**
When selecting an item from the language menu, users cannot tell how many languages are available, nor which language is selected once they are in the menu.

**Steps to Reproduce**
1. Open a Docs document
2. Enable a screen reader
3. Use the keyboard to navigate to the language menu
4. Try to change the language and identify which one you have selected...

**Possible Solution**
Use the [APG pattern](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/#:~:text=when%20a%20menuitemcheckbox%20or%20menuitemradio%20is%20checked%2C%20aria-checked%20is%20set%20to%20true) which has the correct behavior: "When a [menuitemcheckbox](https://w3c.github.io/aria/#menuitemcheckbox) or [menuitemradio](https://w3c.github.io/aria/#menuitemradio) is checked, [aria-checked](https://w3c.github.io/aria/#aria-checked) is set to `true`

There is an example at [this address](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-editor/) with a codepen.

Some [additional information on this page.](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-editor/#:~:text=of%20menu%20items.-,menuitemradio,-li)

**Additional context/Screenshots**
<img width="172" height="259" alt="Image" src="https://github.com/user-attachments/assets/8cf7c0b3-7132-4ede-94c9-1d976a1dd3d1" />

**Comment by Vincentparisgithub** [comment]
i confirm this bug
Whatever the language you choose, the corrector is always in french
